sdks/java/io/iceberg/src/main/java/org/apache/beam/sdk/io/iceberg/RecordWriter.java

```diff
@@ -92,7 +96,15 @@ public long bytesWritten() {
     return icebergDataWriter.length();
   }
 
-  public DataFile dataFile() {
-    return icebergDataWriter.toDataFile();
+  public ManifestFile getManifestFile() throws IOException {
+    String manifestFilename = FileFormat.AVRO.addExtension(absoluteFilename + ".manifest");
```

Comment:
I deliberately want the double extension. Or at least for the name to have `manifest` in it. And the extension is required by Iceberg because it detects filetype by extension in the decode path.